### PR TITLE
Always remove the RO/RW tag from VDIs in case of failure

### PR DESCRIPTION
**Pull request by Wescoeur**
During VDI activation in the blktap module and in case of failure in "sm.VDI.from_uuid" call, the RO/RW tag is never removed. As a result a VDI can no longer be used correctly: an assert is triggered each time we try to re-activate this volume because the tag is still present.

### CODE REVIEW

**Review by MarkSymsCtx — APPROVED** [review]

File: drivers/blktap2.py [hunk 1]
```diff
@@ -1615,7 +1617,6 @@ def _activate_locked(self, sr_uuid, vdi_uuid, options):
             driver_info = target.sr.srcmd.driver_info
```
Comment (rdn32) [inline drivers/blktap2.py:1617]: These few lines of code used to be conditional on `self.tap_wanted()` but now they're run unconditionally. Is that deliberate? Or an accidental side-effect of the way you've moved the beginning of the `try` block?

Comment (MarkSymsCtx) [reply drivers/blktap2.py:1617]: well spotted, that's a bug

**Review by MarkSymsCtx — CHANGES_REQUESTED** [review]

File: drivers/blktap2.py (hunk 1 above)
Comment (Wescoeur) [reply drivers/blktap2.py]: I pushed this change a little too quickly... So fixed I guess this time. :)

**Review by rdn32 — APPROVED** [review]

**Comment by MarkSymsCtx** [comment]
Being picky it's not actually the `pause tag` you're ensuring is removed by rather the active on host RW/RO key. The change itself looks fine and I'm not overly bothered if the commit message is a bit misleading.

**Review by MarkSymsCtx — APPROVED** [review]

**Comment by Wescoeur** [comment]
Right. I modified the description.

**Review by TimSmithCtx — APPROVED** [review]

**Pull request merged**